core/templates/dev/head/pages/landing/fractions/Fractions.js

```diff
@@ -17,8 +17,17 @@
  */
 
 oppia.controller('Fractions', [
-  '$scope', 'UrlInterpolationService',
-  function($scope, UrlInterpolationService) {
+  '$scope', '$timeout', '$window', 'UrlInterpolationService',
+  function($scope, $timeout, $window, UrlInterpolationService) {
     $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+    $scope.getStaticSubjectImageUrl = function(subjectName) {
+      return UrlInterpolationService.getStaticImageUrl('/subjects/' +
+        subjectName + '.svg');
+    };
+    $scope.onClickGetStartedButton = function() {
+      $timeout(function() {
+        $window.location = '/explore/ereONiVLp-VE';
```

Comment:
Normally I've seen this when we have a hook for siteAnalyticsService. It does seem like a good idea to add a call to that, actually. Can we do that? Just declare a new custom event type.

Reply:
Also this seems like the wrong link. I suggest /collection/4UgTQUc1tala

Reply:
Done